patterns/ink/events.ts

```diff
@@ -1,5 +1,19 @@
 import { AccountIdRune, ApplyExtrinsicEventPhase, Rune, RunicArgs } from "../../mod.ts"
 
+export interface ContractsRuntimeEvent {
+  phase: ApplyExtrinsicEventPhase
+  event: {
+    type: "Contracts"
+    value: {
+      contract?: Uint8Array
```

Comment:
Not every contract runtime event has the `contract` attribute

See https://github.com/paritytech/substrate/blob/ae83a672f2bbe6a4f7391f6a5b2b6fd7ad4f8651/frame/contracts/src/lib.rs#L747-L816